getting-started-openapi-to-graphql.html

```diff
@@ -1,3 +1,9 @@
+---
+layout: blank
+permalink: /getting-started-openapi-to-graphql.html
+redirect_from: /getting-started-oasgraph
```

Comment:
i think /getting-started-oasgraph never existed.  Should it be `/getting-started-oasgraph.html`?

Reply:
This way both `/getting-started-oasgraph` and `/getting-started-oasgraph.html` work